odl/contrib/solvers/operator/oputils.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2014-2020 The ODL contributors
```

Comment:
I would have preferred a better name for this folder. Perhaps something involving proximal?

Reply:
I mirrored the original layout since we do that in other places, too. But sure, I'll give the module a better name.